runtime/caml/config.h

```diff
@@ -206,17 +206,10 @@ typedef uint64_t uintnat;
    This must be at least [Max_young_wosize + 1]. */
 #define Minor_heap_min (Max_young_wosize + 1)
 
-/* There may be at most 1<<Minor_heap_sel_bits minor
-   heaps allocated */
-#define Minor_heap_sel_bits 7
-
-/* An entire minor heap must fit inside one region
-   of size 1 << Minor_heap_align_bits, which determines
-   the maximum size of the heap */
 #if SIZEOF_PTR  <= 4
```

Comment:
Trunk has
```C
/* Maximum size of the minor zone (words).
   Must be greater than or equal to [Minor_heap_min].
*/
#define Minor_heap_max (1 << 28)
```
`(1<<28)` words is ~2GB per minor heap on a 64bit machine, I think we should just take that.